### PR TITLE
Sample fetch version

### PR DESCRIPTION
**Pull request by mary-dcouto**
sample program to fetch ziti sdk version and the ziti controller version

### CODE REVIEW

File: Ziti.NET.Standard/Ziti.NET.Standard.csproj [hunk 1]
```diff
@@ -39,6 +39,7 @@
         <PackageReference Include="System.Memory" Version="4.5.4" />
         <PackageReference Include="System.Text.Json" Version="6.0.4" />
         <PackageReference Include="Ziti.NET.Standard.native" Version="0.28.1.76" />
+        <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```
Comment (dovholuknf) [inline Ziti.NET.Standard/Ziti.NET.Standard.csproj:42]: I'd rather not introduce another dependency if we can avoid it

File: Ziti.NET.Standard/src/OpenZiti/ZitiIdentity.cs [hunk 2]
```diff
@@ -511,6 +525,16 @@ public void ZitiResponse(object evt)
 		public void ZitiDumpToFile(string fileName) {
 			NAPI.z4d_ziti_dump_file(this.WrappedContext.nativeZitiContext, fileName);
 		}
+
+		private async Task<string> GetControllerVersionAsync(string ztAPI)
+		{
+			string versionUrl = "version";
+			string versionReponse = await OpenZiti.WebClient.HttpGet(ztAPI, versionUrl);
```
Comment (dovholuknf) [inline Ziti.NET.Standard/src/OpenZiti/ZitiIdentity.cs]: this is already captured for us in the C SDK I thought? Why make a web request?

Comment (dovholuknf) [reply Ziti.NET.Standard/src/OpenZiti/ZitiIdentity.cs]: `ziti_get_controller_version` that should return this information. Is it not working when you use it?

Comment (mary-dcouto) [reply Ziti.NET.Standard/src/OpenZiti/ZitiIdentity.cs]: I will test this function and let you know

**Pull request merged**